terraform/gcp/variables.tf

```diff
@@ -1,4 +1,4 @@
 variable "project"            {}
 variable "region"             {}
 variable "zone"               {}
-variable "ubuntu_1604_image"  { default = "ubuntu-os-cloud/ubuntu-1604-lts" }
+variable "ubuntu_image"  { default = "ubuntu-os-cloud/ubuntu-1804-lts" }
```

Comment:
Note that the GIFT PPA doesn't work with 18.04 yet. Or at least that used to be the case.